hub.go

```diff
@@ -210,7 +210,9 @@ func (hub *Hub) CaptureEvent(event *Event) *EventID {
 	if client == nil || scope == nil {
 		return nil
 	}
-	return client.CaptureEvent(event, nil, scope)
+	eventID := client.CaptureEvent(event, nil, scope)
+	hub.lastEventID = eventID
```

Comment:
Yikes, it is not, separate hubs are required for concurrent code

Reply:
Yeah, so storing the `lastEventID` in a Hub and cloning the hub in a different goroutine is racy.

In [integrations the call to sentry.CurrentHub().Clone()](https://github.com/rhcarvalho/sentry-go/blob/54d52224adc5e7cba27907d9193de18724f4ee59/echo/sentryecho.go#L58) happens in a different goroutine than where the currentHub belongs, and that parent goroutine could be concurrently sending events.

Need to step back and think, perhaps the way forward is to deprecate `sentry.LastEventID()` as it provides no value (?) in Go, plus hurts transport throughput if we preserve correctness.

Talked with @HazAT for context. The `LastEventID` exists historically because of supporting the [user feedback dialog](https://docs.sentry.io/enriching-error-data/user-feedback/) in JavaScript when there is a captured exception that would otherwise have no `eventID` associated with it.

Reply:
> *separate hubs are required for concurrent code*

:+1: 

Correct usage of the hub would not lead to races on `lastEventID`.